src/resource/Resource.ts

```diff
@@ -190,6 +194,14 @@ export default abstract class Resource {
     return getEntitySchema(this);
   }
 
+  /** Get the entity schema defining  */
+  static getRequestOptions<T extends typeof Resource>(this: T) {
+    return {
```

Comment:
seems like this method itself is sufficient without needing properties as well. (thus reducing api surface). Then we could just return undefined here by default which would maybe be preferable.

Reply:
What do you think: 
```typescript
// static method:
static getRequestOptions<T extends typeof Resource>(this: T): RequestOptions | undefined;

// or static field:
static requestOptions?: RequestOptions;
```

Reply:
Also, perhaps this typed options object should be passed on as is in the meta of the [FetchAction](https://github.com/coinbase/rest-hooks/pull/11/files#diff-39f8fa22b8033e26276f0093acfaa3d4R43) for consistency

Reply:
it should be a function so it can be resolved polymorphically - in case it's called on its own and not by one of the other requestshape generators

Reply:
since it's called in singleRequest() i'm not sure how you'd have the meta to pass it; or you mean what singleRequest ends up returning?

Reply:
I was thinking of the fetch action dispatched in useFetcher. dataExpiryLength and errorExpiryLength are destructured from the options object and passed as plain properties in the action meta.

Since the RequestShape was changed to have an options property, perhaps it would be more consistent to pass that object on directly instead of destructuring.

Reply:
Oh sure; so like `meta: { options?: { dataExpiryLength?: ....` that sounds smart